schema/spacewalk/update-messages.txt

```diff
@@ -2,19 +2,16 @@
 SUSE Manager Database Schema Update
 
 This patch updates the database schema.
-Additional configuration steps are required.
+The patch will be applied automatically at the end
+of the update.
+In case of a failure, spacewalk services are stopped and cannot
```

Comment:
Shouldn't  it be `spacewalk service` as we had before?

Reply:
Well, there not this one spacewalk service. spacewalk-service is a systemd target and contains multiple services.